client-lib/java/src/main/java/com/abcxyz/jvs/JvsClient.java

```diff
@@ -34,52 +36,80 @@
 @RequiredArgsConstructor(access = AccessLevel.PACKAGE)
 @Slf4j
 public class JvsClient {
-  // This postfix is added by the cli tool when creating breakglass tokens.
-  private static final String UNSIGNED_POSTFIX = ".NOT_SIGNED";
+  // This is the HMAC key to use for creating breakglass tokens. Breakglass
+  // tokens are already "unverified", so having this static secret does not
+  // introduce additional risk, and breakglass is disabled by default.
+  protected static final String BREAKGLASS_HMAC_SECRET =
+      "BHzwNUbxcgpNoDfzwzt4Dr2nVXByUCWl1m8Eq2Jh26CGqu8IQ0VdiyjxnCtNahh9";
 
-  // this category is added by the cli tool when creating breakglass tokens.
-  private static final String BREAKGLASS_CATEGORY = "breakglass";
+  // This is the justification category set for breakglass tokens.
+  private static final String BREAKGLASS_JUSTIFICATION_CATEGORY = "breakglass";
 
   private final JwkProvider provider;
   private final boolean allowBreakglass;
 
-  public DecodedJWT validateJWT(String jwtString) throws JwkException {
-    DecodedJWT jwt = JWT.decode(jwtString);
+  /**
+   * This parses the given token as a breakglass token. If the token is not a JWT, not HS256, or
+   * missing breakglass justifications, it returns null. If the token fails to verify, it throws an
+   * exception.
+   */
+  private DecodedJWT parseBreakglassToken(String tokenStr) throws JwkException {
```

Comment:
So, this is a bit of a nit, but the practice of returning null as a "fail" is not super prevalent within java. With the name of this method, and the fact it shows that it throws a "JwkException" i would expect that i would get that exception if i passed this a non breakglass token. 

The more idiomatic way, in my mind, is to move the breakglass check to the validateJWT method, and only call parseBreakglassToken it it passes. 
(specifically the below moved) 

```
    if (!Header.JWT_TYPE.equals(token.getType())
        || !SignatureAlgorithm.HS256.getValue().equals(token.getAlgorithm())) {
      log.debug("token is not breakglass");
      return null;
    }
```

Reply:
oh, i see it used to be similiar to that with a separate method to check that it was valid and allowed. Why did we refactor that out?

Reply:
I think the logic here doesn't regard a non-breakglass token as an error. It's like "not breakglass I don't care"

Reply:
Well, given the name is "parseBreakglass" i would expect an error if i passed it a non breakglass token.

Reply:
I switched to to `validateBreakglassToken` and moved the header check into validateJWT.